apps/web/pages/event-types/[type].tsx

```diff
@@ -915,6 +916,9 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                       locations,
                       ...input
                     } = values;
+
+                    if (requirePayment) input.currency = currency;
```

Comment:
There's no "currency" field I the form, hence `input.currency` doesn't exists.

Reply:
Yes, what I do here is to add this field to the `input` object so that it can be sent to the mutation.

Reply:
What I mean is that there's no "currency" value being returned from the form, hence it's generating a type error.

Reply:
Well, I think with the changes applied on this commit https://github.com/calcom/cal.com/pull/2404/commits/23b6d1956b21ca1af6809bb520b4c944bba7eaa3, there is no type error.